content/registry/go-opentracing-go-redis

```diff
@@ -0,0 +1,18 @@
+---
+title: opentracing-go-redis
+registryType: instrumentation
+tags:
+    - golang
+    - redis
+    - go-redis
+repo: https://github.com/lyb0307/opentracing-go-redis
+license: WTFPL
```

Comment:
you may want to consider a different, more widely used license, like MIT or APLv2.

Reply:
I have changed it to MIT. Thank you for the reply~